### PR TITLE
[WIP] [CI:DOCS] Add Configuration section to readthedocs

### PR DESCRIPTION
**Pull request by rhatdan**
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by rhatdan** [comment]
@TomSweeneyRedHat @baude @vrothberg PTAL

One issue with this is figuring out how to get the links to work correctly.

**Comment by rhatdan** [comment]
@edsantiago Not sure if you know of anything in readthedocs, but PTAL

File: docs/source/Configuration.rst [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.. include:: includes.rst
+
+Configuration Files
+========
+
+:doc:`containers.conf <markdown/containers.conf.5>` The containers configuration file specifies all of the available configuration command-line options/flags for container engine tools like Podman & Buildah, but in a TOML format that can be easily modified and versioned.
+
+:doc:`policy.json <markdown/containers-policy.json.5>` Signature verification policy files are used to specify  policy,  e.g. trusted  keys, applicable when deciding whether to accept an image, or individual signatures of that image, as valid.
+
+:doc:`podman-systemd.unit <markdown/podman-sysemd.unit.5>` systemd units using Podman Quadlet
```
Comment (edsantiago) [inline docs/source/Configuration.rst:10]: Dunno anything about readthedocs but I'm guessing that `sysemd` (here and line 12) are typos, missing 't'?

**Comment by Luap99** [comment]
Note to test this out locally you can follow these steps: https://github.com/containers/podman/tree/main/docs#local-testing.

File: docs/source/Configuration.rst [hunk 2]
```diff
@@ -0,0 +1,18 @@
+.. include:: includes.rst
+
+Configuration Files
+========
+
+:doc:`containers.conf <markdown/containers.conf.5>` The containers configuration file specifies all of the available configuration command-line options/flags for container engine tools like Podman & Buildah, but in a TOML format that can be easily modified and versioned.
+
+:doc:`policy.json <markdown/containers-policy.json.5>` Signature verification policy files are used to specify  policy,  e.g. trusted  keys, applicable when deciding whether to accept an image, or individual signatures of that image, as valid.
```
Comment (TomSweeneyRedHat) [inline docs/source/Configuration.rst:8]:
```suggestion
:doc:`policy.json <markdown/containers-policy.json.5>` Signature verification policy files are used to specify policy,  e.g., trusted keys, applicable when deciding whether to accept an image or individual signatures of that image as valid.
```

**Comment by TomSweeneyRedHat** [comment]
Couple nits.  The setup look OK to me, but the linter isn't buying it at all. Something is off.  I'd ping @bbaude next week

**Comment by vrothberg** [comment]
ping

**Review by vrothberg — COMMENTED** [review]
@rhatdan still working on this PR?

**Comment by rhatdan** [comment]
I have no clue how to make this work, so closing.

**Pull request closed without merging**